opm/parser/eclipse/EclipseState/Schedule/MSW/WellSegments.hpp

```diff
@@ -109,23 +98,13 @@ namespace Opm {
 
     private:
         void processABS();
-        void processINC(const bool first_time);
-
-        std::string m_well_name;
-        // depth of the nodal point of the top segment
-        // it is taken as the BHP reference depth of the well
-        // BHP reference depth data from elsewhere will be ignored for multi-segmented wells
-        double m_depth_top;
-        // length down the tubing to the nodal point of the top segment
-        double m_length_top;
-        // effective wellbore volume of the top segment
-        double m_volume_top;
-        // type of the tubing length and depth information
-        LengthDepth m_length_depth_type;
+        void processINC(double depth_top, double length_top);
+        void process(LengthDepth length_depth, double depth_top, double length_top);
+        void addSegment(const Segment& new_segment);
+        void loadWELSEGS( const DeckKeyword& welsegsKeyword);
+
         // components of the pressure drop to be included
         CompPressureDrop m_comp_pressure_drop;
-        // multi-phase flow model
-        MultiPhaseModel m_multiphase_model;
```

Comment:
Some members are not used at the moment, while we might need to add them back later.  It is okay to be removed for now.